data/human/ships.txt

```diff
@@ -435,13 +435,13 @@ ship "Bastion"
 	attributes
 		category "Medium Warship"
 		"cost" 3860000
-		"shields" 7500
-		"hull" 4700
+		"shields" 9000
+		"hull" 5500
 		"required crew" 17
 		"bunks" 40
-		"mass" 1210
-		"drag" 16.5
-		"heat dissipation" .32
+		"mass" 950
+		"drag" 12.8
+		"heat dissipation" .4
```

Comment:
```suggestion
		"mass" 1100
		"drag" 15.5
		"heat dissipation" .34
```
This is equivalent to going back and giving the Bastion a multiplier of 1.5 in the mass rework (rather than the 1.6 it recieved) - that is, this preserves its pre-0.10.7 handling.